opendevin/runtime/client/client.py

```diff
@@ -254,7 +248,7 @@ async def run_ipython(self, action: IPythonRunCellAction) -> Observation:
             )
 
     def get_working_directory(self):
-        result, exit_code = self._execute_bash('pwd', keep_prompt=False)
+        result, exit_code = self._execute_bash('pwd', timeout=60, keep_prompt=False)
```

Comment:
Same here and other timeouts, if they are the timeout of a bash command, that the user can define in sandbox_config.timeout, then we could use that default?

Reply:
For this one - this is actually part of client initialization, so we hard code the timeout. I'll add a comment to clarify!